helm/trivy/templates/statefulset.yaml

```diff
@@ -17,7 +17,9 @@ spec:
       app.kubernetes.io/instance: {{ .Release.Name }}
   {{- if .Values.persistence.enabled }}
   volumeClaimTemplates:
-    - metadata:
+    - apiVersion: v1	
+      kind: PersistentVolumeClaim
```

Comment:
If I understand correctly from [the docs](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#volume-claim-templates),  [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) is created by default. so this update doesn't affect on k8s deploying. 

But it seems ArgoCD tries to compare the state in the cluster with what is specified, it notices that the kind and apiVersion are not specified in one place, and it considers this a discrepancy, which leads to an "out of sync" state.

So I think It makes sense.

Reply:
Also it's useful if the k8s behavior will be changed.